gravitee-exchange-controller/gravitee-exchange-controller-websocket/src/main/java/io/gravitee/exchange/controller/websocket/server/WebSocketControllerServerVerticle.java

```diff
@@ -38,17 +39,19 @@ public class WebSocketControllerServerVerticle extends AbstractVerticle {
 
     private final HttpServer controllerWebSocketHttpServer;
     private final WebSocketRequestHandler webSocketRequestHandler;
+    private final FailureHandler failureHandler;
 
     @Override
     public Completable rxStart() {
         Router router = Router.router(vertx);
-        router.route(EXCHANGE_CONTROLLER_PATH).handler(webSocketRequestHandler);
+        router.route(EXCHANGE_CONTROLLER_PATH).handler(webSocketRequestHandler).failureHandler(failureHandler);
```

Comment:
We could directly instantiate the failure handler here, it is not realy to controller to decide it.